src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss

```diff
@@ -1,14 +1,18 @@
 @import '../../../common/variables';
 
-.bcs-ActivityThread-toggle {
-    margin-left: $sidebarActivityFeedSpacingHorizontal;
-    color: $bdl-box-blue;
-    font-weight: bold;
-
-    &:hover,
-    &:active {
+.bcs-ActivityThread {
+    .bcs-ActivityThread-toggle {
+        margin-top: -9px;
         margin-left: $sidebarActivityFeedSpacingHorizontal;
+        color: $bdl-box-blue;
         font-weight: bold;
+
+        &:hover,
+        &:active {
+            margin-top: -9px;
```

Comment:
Can we add a brief comment as to why we selected this value? Is it based on another style that could change independently and cause a visual regression?